app/src/org/commcare/gis/DrawingBoundaryActivityUIController.kt

```diff
@@ -0,0 +1,71 @@
+package org.commcare.gis
+
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import org.commcare.dalvik.R
+import org.commcare.interfaces.CommCareActivityUIController
+import org.commcare.utils.StringUtils
+import org.commcare.views.ManagedUi
+import org.commcare.views.UiElement
+import org.javarosa.core.services.locale.Localization
+
+@ManagedUi(R.layout.activity_drawing_boundary)
```

Comment:
It's awesome that decorators still work the same way across the kotlin files

Reply:
yes! I was very skeptical that this would work before trying it out.